extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/channels.xml

```diff
@@ -252,4 +252,18 @@
 		<state min="0" max="68399" step="1" readOnly="false">
 		</state>
 	</channel-type>
+
+	<channel-type id="shuffle">
+		<item-type>Switch</item-type>
+		<label>Shuffle</label>
+		<description>Shuffle queue playback</description>
+		<category>Switch</category>
```

Comment:
All channels of type Switch in this binding have the switch category set. Should we suppress the category more generally for the channels of this binding ?